### PR TITLE
3760 checkout fixes

### PR DESCRIPTION
**Pull request by Mopok**
Do #22  first.

Product checkout fixes

### CODE REVIEW

File: partials/shop-invoicetotals.htm [hunk 1]
```diff
@@ -5,29 +5,25 @@
     {% set cartTotalNoTax = cartTotalNoTax + item.price %}
   {% endfor %}
 {% endfor %}
-{% set discount = totals.discount_total %}
+{% set discount = invoice.total_discount %}
 {% if cartTotalNoTax > 0 %}
     <li class="list-group-item">Item Total: <div class="pull-right">{{ cartTotalNoTax|currency }}</div></li>
```
Comment (tomcornall) [inline partials/shop-invoicetotals.htm]: Reminder for the total here, should say only "Total:" (if it is not an item total)

File: pages/checkout-start/page-checkout-start.htm [hunk 2]
```diff
@@ -41,14 +41,7 @@
             <div class="form group">
             <div class="col-xs-12 col-sm-6">
                 <form class="form-guest-checkout">
-                    <p id="title-p"><strong>Guest Checkout</strong></p>
-                    <a href="{{ site_url('checkout') }}"><span class="btn btn-sm btn-important-special">Continue as a Guest</span></a><br><br>
-                </form>
```
Comment (tomcornall) [inline pages/checkout-start/page-checkout-start.htm:46]: No more guest checkout - is this desired?

File: partials/shop-cart-totals.htm [hunk 3]
```diff
@@ -1,33 +0,0 @@
...
-				{% endif %}
-		</div>
-
-	{% else %}
-		<h4 class="md-bordered-title">Your cart is empty!</h4>
-		<a class="md-button col-xs-12 cart-empty-button" href="{{ site_url('shop') }}">Continue shopping <span class="fa fa-arrow-circle-right"></span></a>
-	{% endif %}
-	</div>
-</div>
-</div>
```
Comment (tomcornall) [inline partials/shop-cart-totals.htm:33]: The cart totals are directly in the cart page now? That's ok if so, just want to be sure this partial was meant to be deleted.

File: partials/shop-invoicetotals.htm [hunk 4]
```diff
@@ -5,29 +5,25 @@
...
+    {% set subDiscount = (invoice.order.subscription.billingPlan.discount_percentage|trim('%') * (cartTotalNoTax + fee - invoice.total_discount) / 100) %}
+    {% set discount = totals.total_discount + subDiscount %} 
 {% endif %}
 {% if invoice.total_shipping_quote > 0 %}
     <li class="list-group-item">Shipping: <div class="pull-right">{{ invoice.total_shipping_quote|currency }}</div></li>
 {% endif %}
-    {% if invoice.total_tax > 0 %}
-<li class="list-group-item">Tax:<div class="pull-right">{{ invoice.total_tax|currency }}</div></li>
+    {% if invoice.tax_total > 0 %}
+<li class="list-group-item">Tax:<div class="pull-right">{{ invoice.tax_total|currency }}</div></li>
```
Comment (tomcornall) [inline partials/shop-invoicetotals.htm:28]: 🌮

**Review by tomcornall — CHANGES_REQUESTED** [review]
@Mopok Some questions, and a reminder to fix the total label

File: partials/shop-cart-totals.htm (hunk 3 above)
Comment (Mopok) [reply partials/shop-cart-totals.htm:33]: They now use the same partial from the checkout. Because they use the same variables.

Comment (tomcornall) [reply partials/shop-cart-totals.htm:33]: 👍

**Review by tomcornall — APPROVED** [review]

**Pull request merged**